_delb/nodes.py

```diff
@@ -2782,6 +2782,12 @@ class QueryResults(Sequence[TagNode]):
     def __init__(self, results: Iterator[_ElementWrappingNode]):
         self.__items = cast(Tuple[TagNode], tuple(results))
```

Comment:
very btw, shouldn't this be `Tuple[TagNode, ...]`?

Reply:
I guesz...